pretix_mollie/payment.py

```diff
@@ -152,6 +155,27 @@ def settings_form_fields(self):
                      required=not bool(self.settings.api_key),
                  )),
             ]
+
+        help_text_order_based = format_html(
+            '<span class="fa fa-info-circle"></span> {}',
+            _('This payment method requires pretix to submit additional data to Mollie, including the full invoice '
+              'address and the purchased products. Note that you might need to add this additional data transfer '
+              'to your privacy policy.')
+        )
+        if not self.event.settings.invoice_address_required:
+            help_text_order_based = format_html(
+                help_text_order_based + '<br><span class="fa fa-warning"></span> {}',
+                _('This payment method will be unavailable if no invoice address has been entered, so consider making '
+                  'this required.'),
+            )
+        help_text_order_based_financing = format_html(
+            help_text_order_based + '<br><span class="fa fa-warning"></span> {}',
+            _('This payment method includes a financing component, i.e. the user has to pay after the services have '
+              'been delivered. pretix will mark the order as "delivered" right after the payment is received since '
```

Comment:
Is it right after the payment is received or more along the lines of "after the financing request has been approved"?

Reply:
Uhm, I was assuming them to approve those requests in real time. I'll change it to "payment has been started" for clarity